doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_smp.rst

```diff
@@ -0,0 +1,460 @@
+| Installation/Boot           | Limited by static       | Fully customizable through SUIT  |
+| Customization               | image metadata          | manifest directives              |
++-----------------------------+-------------------------+----------------------------------+
+| State information           | Embedded within the     | Update candidate existence is    |
+|                             | image slot metadata     | represented within the SUIT      |
+|                             |                         | manifest.                        |
++-----------------------------+-------------------------+----------------------------------+
+
+SUIT in SMP image management group
+==================================
+
+The SMP protocol can provide SMP-based SUIT image management, assuming that a SUIT-based system, even a complex one, can be represented as a *single image* system.
+In this model, the contents of the *primary slot* represent the currently installed SUIT root manifest, while the *secondary slot* holds the update candidate root manifest.
+
+Get State of Images Request
+---------------------------
+
+This command retrieves images and their states.
+The response is in CBOR format:
+
+.. code-block:: cbor
+
+    {
+    (str)"images" : [
+        {
+            (str,opt)"image"      : (int)       //SUIT - always 0 as system would be seen as "single image" system
+            (str)"slot"           : (int)       //SUIT - 0 - "primary" - installed system, 1 - update candidate in DFU partition
+            (str)"version"        : (str)       //SUIT - sequence-number from root manifest.
+            (str)"hash"           : (byte str)  //SUIT - digest of root manifest.
+            (str,opt)"bootable"   : (bool)      //SUIT - root manifest always bootable.
+            (str,opt)"pending"    : (bool)      //SUIT - true when info about update candidate is delivered to SDFW - can be obtained from SDFW SUIT_Storage
+            (str,opt)"confirmed"  : (bool)      //SUIT - no swapping, no rollback after unsuccessful update.
+            (str,opt)"active"     : (bool)      //SUIT - has no sense for the manifest - could make a sense for component when it supports A/B images
+            (str,opt)"permanent"  : (bool)      //SUIT - no swapping, no rollback after unsuccessful update.
+        }
+        ...
+    ] (str,opt)"splitStatus" : (int) }
+
+Image Upload Request
+--------------------
+
+This command uploads an image to the device.
+Payloads for all SUIT components are delivered as integrated payloads.
+The CBOR data format is:
+
+.. code-block:: cbor
+
+    {
+        (str,opt)"image"    : (uint)     //SUIT - always 0 as system would be seen as "single image" system
+        (str,opt)"len"      : (uint)
+        (str)"off"          : (uint)
+        (str,opt)"sha"      : (str)
+        (str,opt)"data"     : (byte str)
+        (str,opt)"upgrade"  : (bool)
+    }
+
```

Comment:
One extra command related to SMP IMG management group shall be documanted as well:

Set state of image request

This command triggers a installation process using SUIT envelope stored in DFU partition. The CBOR data format is:

{ 
    (str)"confirm"      : (bool) 	//Must be stet to 'true'
}